### PR TITLE
Add online sessions flag

### PR DESCRIPTION
**Pull request by patch0**

### CODE REVIEW

File: web/public/js/directives/cd-start-dojo/venue/template.dust [hunk 1]
```diff
@@ -78,6 +78,19 @@
...
+          <div class="cd-simple-flex-grid__flex-1 cd-sad-venue__online">
+            <div class="checkbox-inline">
+              <input type="checkbox" id="online_dojo" ng-model="$ctrl.venue.onlineSessions"
+                name="onlineSessions"
+                ng-false-value="0"
+                ng-true-value="1"
+                ng-checked="$ctrl.venue.onlineSessions == 1">
+              <label for="online_dojo">{@i18n key="Online sessions"/}</input>
+            </div>
+            <p class="help-block">{@i18n key="This Dojo will run online (partially or completely) Check this box if your club runs online all or some of the time."/}</p>
```
Comment (patch0) [inline web/public/js/directives/cd-start-dojo/venue/template.dust:91]: I'd split this into two.

```suggestion
              <label for="online_dojo">{@i18n key="This Dojo will run online (partially or completely)"/}</input>
            </div>
            <p class="help-block">{@i18n key="Check this box if your club runs online all or some of the time."/}</p>
```

File: web/public/js/directives/dojo/dojo-form/status/template.dust [hunk 2]
```diff
@@ -53,6 +53,24 @@
...
+        <div class="col-lg-8">
+          <div class="checkbox cd-checkbox checkbox-primary">
+            <input type="checkbox"
+                  ng-model="dojo.onlineSessions"
+                  id="onlineSessionsCheckbox"
+                  ng-true-value="1"
+                  ng-false-value="0"
+                  ng-change="updateLocalStorage('dojoListing','onlineSessions', dojo.onlineSessions)"
+                >
+                <label class="cd-checkbox-label1" for="onlineSessionsCheckbox">{@i18n key="My club runs online (partially or completely) Check this box if your club runs online all or some of the time."/}</label>
```
Comment (patch0) [inline web/public/js/directives/dojo/dojo-form/status/template.dust:68]:
```suggestion
        <label class="col-lg-4 cd-form-label1 with-checkbox">{@i18n key="My club runs online (partially or completely)"/}</label>
        <div class="col-lg-8">
          <div class="checkbox cd-checkbox checkbox-primary">
            <input type="checkbox"
                  ng-model="dojo.onlineSessions"
                  id="onlineSessionsCheckbox"
                  ng-true-value="1"
                  ng-false-value="0"
                  ng-change="updateLocalStorage('dojoListing','onlineSessions', dojo.onlineSessions)"
                >
                <label class="cd-checkbox-label1" for="onlineSessionsCheckbox">{@i18n key="Check this box if your club runs online all or some of the time."/}</label>
```

**Review by patch0 — COMMENTED** [review]
I think this looks OK (although I can't approve because Github)

Splitting the text into two might make it clearer.  Failing that you need to put a full stop between the two parts :)

**Comment by glenpike** [comment]
Edit Dojo
Screenshot 2020-10-15 at 09 52 42

Start Dojo
Screenshot 2020-10-15 at 09 53 13

**Comment by patch0** [comment]
LGTM :+1:

**Pull request merged**